src/main/java/frc/robot/RobotContainer.java

```diff
@@ -72,8 +85,8 @@ private void configureButtonBindings() {}
    */
   public Command getAutonomousCommand() {
     // An ExampleCommand will run in autonomous
-    DriveConstantControlCommand x = new DriveConstantControlCommand(SwerveDriveSystem, primaryController);
-    SwerveDriveSystem.setDefaultCommand(x);
-    return x;
+  //  DriveConstantControlCommand x = new DriveConstantControlCommand(SwerveDriveSystem, primaryController);
+    // SwerveDriveSystem.setDefaultCommand(x);
+    return new UseArm(primaryController, transport);
```

Comment:
I think we'll have to rewrite this when we merge the branches anyway, even though this code isn't correct.